web-client/public/js/setup-handlers.js

```diff
@@ -5,7 +5,11 @@ import {
     GREY_EDGES_DASHED_SIDEBAR
 } from "./constants";
 
+import { setupLoadAndImportHandlers } from "./setup-load-and-import-handlers";
```

Comment:
We started by putting all of the load and import code right in _setup-handlers_ (see the individual commits), but when the main work was done, it made sense to separate them into their own module. Thus, the details are hidden away unless someone wants to look into that file, and here you only need the single high-level setup function.